cilium/privileged_service_client.cc

```diff
@@ -10,8 +10,8 @@ namespace PrivilegedService {
 
 ProtocolClient::ProtocolClient()
     : Protocol(CILIUM_PRIVILEGED_SERVICE_FD), call_mutex_(PTHREAD_MUTEX_INITIALIZER), seq_(0) {
-  RELEASE_ASSERT(get_capabilities(CAP_EFFECTIVE) == 0 && get_capabilities(CAP_PERMITTED) == 0,
-                 "cilium-envoy running with privileges, exiting");
+  /* RELEASE_ASSERT(get_capabilities(CAP_EFFECTIVE) == 0 && get_capabilities(CAP_PERMITTED) == 0, */
```

Comment:
Double-Check dropped capabilities in Envoy process -> currently commented out. 

TODO: Re-enable (with enhanced logic) or delete

Reply:
logic re-enabled. only allowed capability is NET_BIND_SERVICE